src/calculateRentalCost.js

```diff
@@ -5,6 +5,15 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  let price = days * 40;
```

Comment:
Consider using a constant for the daily cost instead of the literal '40' to make the code more maintainable and self-explanatory.